bundle/terranova/plan.go

```diff
@@ -134,10 +97,68 @@ func CalculateDeployActions(ctx context.Context, b *bundle.Bundle) ([]deployplan
 		}
 	}
 
+	// We're processing resources in DAG order, because we're trying to get rid of all references like $resources.jobs.foo.id
+	// if jobs.foo is not going to be (re)created. This means by the time we get to resource depending on $resources.jobs.foo.id
+	// we might have already got rid of this reference, thus potentially downgrading actionType
+
+	// parallelism is set to 1, so there is no multi-threaded access there.
+	err = g.Run(1, func(node nodeKey) {
```

Comment:
Could this run with >1 though?

Reply:
not today, because it e.g. appends to actions